signup-page.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Login | Ecommerce</title>
+    <link rel="stylesheet" href="main.css">
+    <link rel="stylesheet" href="https://socket-ui.netlify.app/components/components.css"/>
+    <script src="https://kit.fontawesome.com/124ff44aed.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header>
+        <nav class="navbar-container">
+
+            <!-- ------------------- Navbar left start ------------------- -->
+            <div class="navbar-left">
+                <div class="hamburger-icon nav-icon-badge">
+                  <i class="fa fa-bars" aria-hidden="true"></i>
+                </div>
+                <div class="nav-logo">
+                  <img
+                    src="./images/logo-ecom.png"
+                    alt="logo"
+                  />
+                </div>
+            </div>
+    
+            <!-- ----------------------- NavBar left end --------------- -->
```

Comment:
Suggestion: You can only write `Navbar` in comment at the start of the code, no need to write start and end line of comments, this will increase your lines.